README.md

```diff
@@ -17,7 +17,7 @@ A aplicação facilitará para usuários conseguirem focar nos seus treinos e pa
 ## Métricas
 
 ## Heroku
-* TODO: Adicionar link
+* URL: https://sin5005-tesi-2023-95e97395cf1b.herokuapp.com/
```

Comment:
Por favor, adicione o badge de status do Heoku, não somente o link.

Caso precise de ajuda, não hesite em perguntar :)

Reply:
Aqui está o link do shield para o Heroku:
https://img.shields.io/badge/Heroku-430098?style=for-the-badge&logo=heroku&logoColor=white

Só seguir o padrão do resto do README (como o ruby, por exemplo) e criar o shield com o link que vc adicionou.

Reply:
Boa dica, corrigido.